doc/api/esm.md

```diff
@@ -1893,11 +1893,11 @@ The resolver can throw the following errors:
 
 > 1. Let _scopeURL_ be _url_.
 > 1. While _scopeURL_ is not the file system root,
```

Comment:
I don't think we've given up on properly expressing this in pseudocode, have we?

Reply:
There's no need for a do-while, the condition holds fine because the first scopeURL is effectively ignored by moving the set up.